### PR TITLE
Added README

### PR DESCRIPTION
**Pull request by Kostaszki**
Added a short README. Setup, License information and the disclaimer are taken from the induction equation README.

### CODE REVIEW

**Comment by Kostaszki** [comment]
Like this?

File: README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Hyperbolic Conservation Laws
```
Comment (ranocha) [inline README.md]: Maybe just "Conservation Laws"?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+[![License](https://licensebuttons.net/l/by-nc-nd/3.0/88x31.png)](https://creativecommons.org/licenses/by-nc-nd/4.0/legalcode)
+
+This is a set of tools for numerically solving different kinds of hyperbolic conservation laws with OpenCL. Currently implemented are linear transport with constant and variable coefficients, the nonlinear magnetic induction equation and ideal MHD. The code is generalised in a form that allows
```
Comment (ranocha) [inline README.md]: Maybe use an unnumbered list for the different (physical) models?

Comment (ranocha) [inline README.md]: A very pedantic person might want to read "(hyperbolic) conservation or balance laws".

Comment (ranocha) [reply README.md]: If we change that,  the change should also be made below.

**Pull request merged**